src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import core.basesyntax.enums.Color;
+
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
The method 'getRandomColor' can be simplified by removing the temporary variable and directly returning the result.